arrow/util.py

```diff
@@ -46,6 +52,14 @@ def is_timestamp(value):
         return False
 
 
+def validate_ordinal(value):
+    """Raise the corresponding exception if `value` is an invalid Gregorian ordinal."""
```

Comment:
No need for the markdown ticks around value!

Reply:
> A few remaining comments, then we should be ready to merge. Let me know once they are addressed @pgogri!

Hi @jadchaar Thank you for your comments! Sorry for the delay. I believe I addressed them all, but please let me know if I missed something.